services/src/main/java/org/keycloak/protocol/ciba/decoupledauthn/HttpAuthenticationChannelProvider.java

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.protocol.ciba.decoupledauthn;
```

Comment:
Like in other places, I think we should remove all references to "decoupled" to reference "authentication channel" instead?

Reply:
@pedroigor I've rename "decoupled" with "authentication channel".